analyzers/src/SonarAnalyzer.SourceGenerators/RegexConstants.cs

```diff
@@ -0,0 +1,26 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2023 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarAnalyzer.SourceGenerators;
+
+public static class RegexConstants
+{
+    public static TimeSpan DefaultTimeout => TimeSpan.FromMilliseconds(100);
```

Comment:
Now that the ITs are failing, it is actually a question if we were too strict with the timeout.

Reply:
I've increased the timeout for now. It will reflect the current status of regular expressions. Long term we might want to improve the regular expressions and reduce the timeout.